repository/errors/auth.go

```diff
@@ -0,0 +1,24 @@
+package errors
+
+import (
+	"errors"
+	"fmt"
+
+	svcerr "release-manager/service/errors"
+
+	"github.com/nedpals/supabase-go"
+)
+
+func ToSvcAuthErr(err error) error {
+	var supabaseErr *supabase.ErrorResponse
+	if errors.As(err, &supabaseErr) {
+		switch supabaseErr.Code {
+		case 401, 404:
+			return fmt.Errorf("%w: %s", svcerr.ErrAuthenticationFailed, err.Error())
```

Comment:
Here I guess you use the `NewAuthErr(message)` function mentioned in the previous comment.